projects/common/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts

```diff
@@ -172,6 +172,7 @@ export class LayerCollectionNavigationComponent implements OnInit, OnChanges, On
 
     protected updateListView(id?: LayerCollectionItemOrSearch): void {
         this.selectedCollection = id ?? {type: 'collection', id: this.collectionId};
+        this.navigateCollection.emit(this.selectedCollection as LayerCollectionItem);
```

Comment:
What does updateListView do in the first place?

Why does it emit something now? (despite the bug)

Reply:
müsste das nicht in die navigate-Methode dann?

Reply:
I tried to separate it a bit better but I don't really want to touch any of the breadcrumb stuff I don't have to 5a41345c365156cb3a8ccbfb6bb41ebf9a4cab2d